consensus/src/consensus_worker.rs

```diff
@@ -1345,10 +1345,15 @@ impl ConsensusWorker {
         }
 
         // add stale blocks to stats
-        let new_stale_block_ids_slots = self.block_db.get_new_stale_blocks();
+        let new_stale_block_ids_creators_slots = self.block_db.get_new_stale_blocks();
         let timestamp = UTime::now(self.clock_compensation)?;
-        for (_b_id, _b_slot) in new_stale_block_ids_slots.into_iter() {
+        for (b_id, (b_creator, b_slot)) in new_stale_block_ids_creators_slots.into_iter() {
             self.stale_block_stats.push_back(timestamp);
+
+            let creator_addr = Address::from_public_key(&b_creator)?;
```

Comment:
Maybe we can add a staking public keys hashset to avoid recomputing an address just for logging

Reply:
it's an option indeed... but it adds a lot of complexity (we need to maintain another index)

if the performance hit is low (which it is), this is OK